runtime/kusama/src/lib.rs

```diff
@@ -79,7 +79,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_name: create_runtime_str!("parity-kusama"),
 	authoring_version: 2,
 	spec_version: 1054,
-	impl_version: 0,
+	impl_version: 1,
```

Comment:
Doesn't seem like anything changed in runtime behavior from substrate side, but would appreciate that someone double checked.

Reply:
Please have this carefully reviewed. If there is a new runtime (wasm blob) with new functions and it does not have a new spec_version id there will be quite a bit of impact in the ecosystem.

Reply:
Blocks would fail to import with the native runtime, the node would print out an error like invalid state root and then the block would still be imported using wasm runtime.

Reply:
Yeah no problem to bump the impl_version as long as we are absolutely certain that the runtime functions and metadata descriptions have not changed.

Reply:
Didn't mean to diminish your comment either, I appreciate your heads up :) We already deployed this PR on some of our nodes an hour ago to test the release.

Reply:
It is actually great that this sort of stuff pops-up in your CI-pipeline :)

Reply:
Everything fine: https://github.com/paritytech/substrate/compare/d383b0dd542bc04d6fd7042205f353cfc76d5502...68e51d6e24862d499b5f042321cc87b172579e74